src/modules/commander/Commander.cpp

```diff
@@ -2324,6 +2324,16 @@ Commander::run()
 				armed.prearmed = (hrt_elapsed_time(&commander_boot_timestamp) > 5_s);
 			}
 
+			/* Lock all actuator output until armed if the circuit breaker is set */
+			if (_param_com_prearm_lock.get()) {
+				if (armed.prearmed && !armed.armed) {
+					armed.lockdown = true;
+
+				} else {
+					armed.lockdown = false;
+				}
+			}
```

Comment:
Do you need an `else` here or is the default `armed.lockdown = false` already?

Reply:
I tried without --thinking that it would be better not to unset the flag if it was set elsewhere-- but it did not work as expected. It is needed otherwise the flag is not unset.

Reply:
Oh, of couse, that makes sense.